.github/workflows/linux.yml

```diff
@@ -105,7 +105,7 @@ jobs:
       run: |
         COVERAGE_BUILD=${HOME}/build
         mkdir --parents ${COVERAGE_BUILD}
-        sg lxd -c "/snap/bin/lxc profile device add default build disk source=${COVERAGE_BUILD} path=/root/parts/multipass/build"
+        sg lxd -c "/snap/bin/lxc profile device add default build disk source=${COVERAGE_BUILD} path=/root/parts/multipass"
         echo "::set-output name=build::${COVERAGE_BUILD}"
```

Comment:
You want to add `/build` to this output, otherwise codecov will scan much more than it needs to.

Reply:
Ok, in iterating over this, I had it there, removed it, added it back, etc. and it never seemed to make a difference.  But if adding is The Right Thing to Do, then I'll put it there.

Reply:
Ok, done and rebased into a previous related commit.